src/autocomplete/Components.tsx

```diff
@@ -75,7 +75,7 @@ export const PillCompletion = forwardRef<IPillCompletionProps, any>((props, ref)
             ref={ref}
         >
             { children }
-            <span className="mx_Autocomplete_Completion_title">{ title }</span>
+            <span className="mx_Autocomplete_Completion_title" dangerouslySetInnerHTML={{ __html: title }}>{ }</span>
```

Comment:
This is not a safe change without extensive documentation - I don't think we can rely on this as-is.